src/core/interpreter.cc

```diff
@@ -1017,9 +997,9 @@ int Interpreter::RedisGenericCommand(bool raise_error, bool async, ObjectExplore
     return 0;
 
   // Raise error for regular 'call' command if needed.
-  if (raise_error && translator->HasError()) {
+  if (*raise_error && translator->HasError()) {
```

Comment:
what does it mean 
```
if (!translator)
    return 0;
```
above ?

Reply:
I actually dont know.. this is not code that I changed

Reply:
I see we use it inside DragonflyHashCommand, were we define a custom explorer StringCollectorTranslator
but why do we return here 0 I dont know

Reply:
I changed it to return 1 and it looks like everything works